cnesreport.properties

```diff
@@ -1,14 +1,19 @@
 category=Visualization/Reporting
 description=CNES plugin that allows users to download a bundle of project reports in multiple formats.
 homepageUrl=https://github.com/lequal/sonar-cnes-report
-archivedVersions=
-publicVersions=3.1.0
+archivedVersions=3.1.0
+publicVersions=3.2.2
 
 defaults.mavenGroupId=fr.cnes.sonarqube.plugins
 defaults.mavenArtifactId=cnesreport
 
+3.2.2.description=Support SonarQube 8 up to 8.2 & add ability to export from a specific branch
+3.2.2.date=2020-03-10
+3.2.2.sqVersions=[7.9,8.2]
```

Comment:
This range should end with `LATEST` unless you specifically know that this version _won't_ be compatible with 8.3. That way when SQ 8.3 is released this version will automatically be marked as compatible with it.

Reply:
Like I said, the previous version is not compatible with SQ 8, and because of the LATEST keyword, we received a lot of issues and we were not able to deal with them fast enough.  
For SQ 8.3, we can't predict if a change would break our plugin.  

So now, we would like to have control on the compatible versions... that's why I ended the range with the last SQ release.  
Is that still a problem ?

Reply:
Totally up to you. Let's leave it at 8.2. AND we still don't want to revert the previous version to `LATEST`.

Reply:
Ok, let's keep 8.2 then.  
And I still won't revert the previous version to LATEST, I promise !